### ISSUE TITLE
youtube ads are no longer being blocked?

### ISSUE THREAD

**Issue by mrawbabylon** [bromite/bromite#1146]
### Bromite version

Version: 90.0.4430.204
Arch: arm64
Android version: 10 QKQ1. 190910.002
Device model: Redmi Note 7 M1901F7H

### Flags

These are the flags changed under `chrome://flags`:

I'm going to be honest I don't understand what this means. I have tab groups on, is this a problem? There was no problem until yesterday and I've had tab groups for a while now.

### Is this bug about the SystemWebView?

Yes/No/**I don't know**

### Is the bug reproducible with latest version?

**Yes**/No

### Can the bug be reproduced with corresponding Chromium version?

Yes/**No**

### Is this bug happening in an incognito tab?
**Yes**/No

### Allow ads for the site; is the bug caused by the adblocker?
Yes/No
? The bug is that the adblocker is not working...

### Is the bug a crash?

Yes, this is the crash dump:
```
<paste here logcat dump>
```

### Describe the bug

In the past, I used bromite for youtube so I could have ads blocked and be able to leave the tab or app. Now, youtube ads are not being blocked. This happened first last night (15th May) and is still a problem. I tried disenabling and re-enabling the adblocker, then I updated my bromite app and it is still occurring. 

I am not the only person with this problem. [Here is a link](https://github.com/bromite/bromite/discussions/1143) to a discussion where two others mentioned having this problem on youtube.

### Steps to reproduce the bug

Steps to reproduce the bug:
1. Go to youtube.com
2. Click on a video that could have an ad
3. There is an ad when the adblocker is on

Do not write "any website", specify which URLs can be used to reproduce the issue.

### Expected behavior

I expected the ad to be blocked by the adblocker.

### Screenshots

![l](https://user-images.githubusercontent.com/84233338/118387083-3e22ea00-b670-11eb-9123-a14ac7f21eee.jpg)
alternative (link): https://64.media.tumblr.com/7844f1c76fbdcafaa4d49cea2d126598/41dd032d9229dad3-4d/s1280x1920/cc6cfe98f6ee838a56e74a28cefbb8ba273925f0.jpg

**Comment by kokukyocho** [comment]
Hello @mrawbabylon!

I am a bot :robot:

Thanks for submitting this issue!
I noticed that it is missing the template, please edit the issue to match the template for either a [bug](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) or a [feature request](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/feature_request.md).

The issue will be automatically re-opened after that.

**Tip**: if you use GitHub in Desktop mode the template will be proposed to you when submitting issues.

**Comment by mrawbabylon** [comment]
what on earth? i followed the template. someone tell me where i went wrong, please

**Comment by csagan5** [comment]
> i followed the template.

@mrawbabylon the template says to answer "Yes" or "No", not to highlight your answer in bold.

**Comment by csagan5** [comment]
Duplicate of #1147

**Comment by mrawbabylon** [comment]
> > i followed the template.
> 
> @mrawbabylon the template says to answer "Yes" or "No", not to highlight your answer in bold.

oh lol thanks for telling me, i'll keep it mind in case i need to make an issue post in the future
